go.mod

```diff
@@ -1,6 +1,8 @@
 module sigs.k8s.io/descheduler
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.1
```

Comment:
should it be `1.22.2` to match the dockerfile??

Reply:
yeah I should have taken this out. it kept complaning about `go mod tidy` if I didn't

Reply:
let me know if it looks correct now